core/frontend/src/quantity-formatting/QuantityFormatter.ts

```diff
@@ -428,6 +440,16 @@ export class QuantityFormatter implements UnitsProvider {
     }
   }
 
+  /**
+   * @internal
+   */
+  public shutdown(): void {
```

Comment:
I find it a bit confusing, that we have a public class, whose initialization and shutdown methods are marked `@internal`, making it somewhat useless. In any case, I think the common practice is to name methods like this `[Symbol.dispose]`.

Reply:
Thanks, I've updated to use [Symbol.dispose]